packages/next-urql/src/rsc.ts

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
```suggestion
import * as React from 'react';
```

Does this still cause trouble in some cases?